style.css

```diff
@@ -0,0 +1,612 @@
+  padding: 20px;
+  margin-top: 20px;
+  background-size: contain;
+  border-top-left-radius: 70px;
+  background-repeat: no-repeat;
+  background-position: bottom -1px right -13px;
+  background-image: url("./images/contact\ form\ background\ shapes.svg");
+  background-color: #6070ff;
+  border: 24px;
+}
+
+.obtainer {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  padding: 10px;
+  color: #253858;
+  display: flex;
+  align-items: center;
+  font-size: 15px;
+  font-weight: 300;
+  background: #f7f7f9;
+  margin: 10px;
+  border-radius: 8px;
+}
+
+.obtainer img {
+  width: 40px;
+  margin-right: 20px;
+  height: auto;
+  max-width: 100%;
+}
+
+.form-heading {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  text-align: center;
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  margin-top: 103px;
+}
+
+.form-par {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-size: 20px;
+  margin: 10px 0;
+  padding: 10px;
+  line-height: 28px;
+  text-align: center;
+  background: #6070ff;
+  border-radius: 10px;
+}
+
+.name {
+  color: #fff;
+  padding: 20px;
+  margin-top: 150px;
+  background-size: contain;
+  border-top-left-radius: 70px;
+  background-repeat: no-repeat;
+  background-position: bottom -1px right -13px;
+  background-image: url("images/contact\ form\ background\ shapes@2x.svg");
+  background-color: #6070ff;
+  font-family: 'Poppins', sans-serif;
+}
+
+.dubbed {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  text-align: center;
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  margin-top: 103px;
+}
+
+#clickin {
+  border: 1px solid #fff;
+  width: 100%;
+  margin: 10px 0;
+  font-size: 17px;
+  padding: 10px;
+  color: #172b4d;
+  line-height: 20px;
+  border-radius: 8px;
+}
+
+#clickin:focus {
+  border: 1px solid #172b4d;
+}
+
+#textbox {
+  color: #172b4d;
+  font-family: 'Poppins', sans-serif;
+  font-size: normal;
+  line-height: 20px;
+  width: 100%;
+  border-radius: 8px;
+  border: 1px solid #fff;
+  min-height: 140px;
+  padding: 10px;
+  margin: 10px 0;
+}
+
+.button_sent {
+  border-radius: 8px;
+  margin: 10px 0;
+  align-self: center;
+  border: 1px solid #fff;
+  color: #6070ff;
+  font-size: 17px;
+  line-height: 24px;
+  padding: 12px 16px;
+}
+
+.button_sent:hover {
+  color: #fff;
+  background: #6070ff;
+}
+
+.details {
+  padding: 8px;
+  margin: 0 auto;
+}
+
+.centerbutton {
+  justify-content: center;
+  align-items: center;
+  display: flex;
+  height: 80px;
+}
+
+#wrapper {
+  max-width: 1050px;
+  margin: 60px auto 60px auto;
+  background-color: #fff;
+}
+
+#inner {
+  display: table;
+  width: 70%;
+  table-layout: fixed;
+}
+
+.about_myself {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-size: 40px;
+  margin-top: 60px;
+  margin-bottom: 20px;
+  font-weight: 700;
+  color: #172b4d;
+  line-height: 52px;
+  position: relative;
+}
+
+@media screen and (min-width: 768px) {

```

Comment:
- [ ] As requested by the previous reviewer, kindly ensure that your page does not have the white margins:

Your design:
<img width="953" alt="image" src="https://user-images.githubusercontent.com/77190203/203509198-9f808554-43dc-4d11-8f3b-9a4ca85ae52c.png">


<img width="953" alt="image" src="https://user-images.githubusercontent.com/77190203/203509309-741c662b-4478-4f50-8d0d-cf78456c4f1a.png">